src/analyzing/Constant_propagation.ml

```diff
@@ -328,6 +320,11 @@ let rec eval env x : svalue option =
         FN (Id (_, { id_svalue = { contents = Some x }; _ })) )
     when is_lang env Lang.Terraform ->
       Some x
+  | DotAccess (_, _, FN (Id (_, { id_svalue = { contents = Some x }; _ }))) ->
+      Some x
+  | N (IdQualified { name_info = { id_svalue = { contents = Some x }; _ }; _ })
+    ->
+      Some x
```

Comment:
As requested in the related PR, could you add a comment here explaining that the svalue is only available here when using the pro engine?

Reply:
done